include/ibm/locks.hpp

```diff
@@ -112,6 +112,9 @@ class Lock
 
     Lock()
     {
+        // Remove the persistent file
+        std::filesystem::remove(fileName);
+        lockTable.clear();
```

Comment:
This change is not needed. This was removed as part of other defects later.
Pls refer https://github.com/ibm-openbmc/bmcweb/commit/b2e7be91fb609fa04439ea83950a45b8d6661fd5 and https://github.com/ibm-openbmc/bmcweb/blob/1030/include/ibm/locks.hpp#L112

Can you also make the trace enhancements as in that PR https://github.com/ibm-openbmc/bmcweb/commit/b2e7be91fb609fa04439ea83950a45b8d6661fd5

Reply:
Done